build.cmd

```diff
@@ -1,13 +1,13 @@
 @echo off
 
-.paket\paket.bootstrapper.exe
+.paket\paket.bootstrapper.exe 5.0.0-beta008
```

Comment:
> And if you reference paket elsewhere, please make it so that we lock it down to a version, so that we're not eternally on a prerelease version.

OK. It's done.